api/reference/1.0/secrets/index.md

```diff
@@ -14,7 +14,8 @@ There are currently three supported secret types denoted in the `type_of` attrib
 
 * Token - this is a simple string of characters known by both systems
 * Username and Password - this is exactly what it sounds like
-* OAuth - these secrets support the OAuth authentication spec.  Event forwarding will ask you for the required information, then will handle the renewal of these tokens for you on a specified interval
+* OAuth2-Client_Credentials - these secrets support the OAuth2 Client Credentials authentication spec.  Event forwarding will ask you for the required information, then will handle the renewal of these tokens for you on a specified interval
+* OAuth2-Google - these secrets support the OAuth2 Google authentication with Adobe Real-Time CDP Event Forwarding app. Event forwarding will handle the renewal of these tokens for you every 30 minutes
```

Comment:
Are these intended to enumerate the possible values of the `type_of` field? If so, I'd suggest we update these to follow the format of:
```
`[VALUE]` - [DESCRIPTION]
```

whee `[VALUE]` is the literal value to be specified.

Examples:
- `token` - a simple string of characters known by both systems
- `username-password` - username and password
- `oauth2-client_credentials` - ...

Reply:
@remear I made the updates.